src/upgrade.cpp

```diff
@@ -175,6 +175,16 @@ ConfigObject<ConfigValue>* Upgrade::versionUpgrade(const QString& settingsPath)
             delete oldFile;
```

Comment:
Unfortinately, I don't see what you mean exactly and how to fix it.
It seems that there are two declarations of QFile* oldFile, one on line 91 and another one indise #ifdef __APPLE__ or #ifdef __WINDOWS__
could this be a problem ?

I need some help for this one.